src/types.ts

```diff
@@ -9,6 +9,9 @@ export type Control = {
   /** Human-readable description of what this control addresses */
   description: string
 
+  /** Human-readable remarks about this control, (e.g., assessment objectives met by the control) */
+  remarks: string
```

Comment:
```suggestion
  remarks?: string
```

Can we make this optional. I don't know if it is going to be necessary each time

Reply:
What are your thoughts?

Reply:
we will leave it mandatory until we get some feedback